types/node/v16/test/module.ts

```diff
@@ -53,8 +53,8 @@ const entry: Module.SourceMapping = smap.findEntry(1, 1);
 {
     const importmeta: ImportMeta = {} as any; // Fake because we cannot really access the true `import.meta` with the current build target
     importmeta.url; // $ExpectType string
-    importmeta.resolve!("local", "/parent"); // $ExpectType Promise<string>
-    importmeta.resolve!("local", new URL("https://parent.module")); // $ExpectType Promise<string>
```

Comment:
These tests can be moved to files that are only loaded by the non-DOM tsconfig, fwiw.